scopes/html/html.env.ts

```diff
@@ -0,0 +1,12 @@
+import { Environment } from '@teambit/envs';
+
+export class HtmlEnv implements Environment {
```

Comment:
override the getDependencies funciton. you probably don't need all the react deps for a framework less component